packages/core/src/storybook/components/canvas-wrapper/CanvasWrapper.module.scss

```
@@ -18,7 +18,8 @@
   }
 
   & :global(.cm-editor) {
-    padding: var(--sb-spacing-medium);
+    padding: 0 0 0 var(--sb-spacing-medium);
```

can you please change that one to
```suggestion
    padding: var(--sb-spacing-medium);
    padding-right: 0;
```
we want the rest of the padding, only the right padding is now an issue (because of the scrollbar)